config/template.config

```diff
@@ -10,19 +10,22 @@ params {
     // Output location
     output_dir = ""
 
-    // Choices: ["HaplotypeCaller", "Mutect2", "Strelka2", "SomaticSniper", "Muse2", "Delly2-gSV", "Delly2-sSV"]
-    variant_caller = ""
-
     // Choices: ["GRCh37ToGRCh38", "GRCh38ToGRCh37"]
     liftover_direction = ""
 
+    // Choices: ["HaplotypeCaller", "Mutect2", "Strelka2", "SomaticSniper", "Muse2", "Delly2-gSV", "Delly2-sSV"]
+    variant_caller = ""
+
     // Path to pre-trained random forest model
     rf_model = ""
 
     // Path to reference fasta files
     fasta_ref_37 = "" // GRCh37-EBI-hs37d5/hs37d5.fa
     fasta_ref_38 = "" // GRCh38-BI-20160721/Homo_sapiens_assembly38.fasta
 
+    // Path to Funcotator data source directory containing dbSNP, GENCODE and HGNC sources for SNV annotation
+    funcotator_data_source = ""
+
```

Comment:
**suggestion**: I suggest adding this parameter to `config/schema.yaml` to make sure it gets validated